src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,452 @@
+package duke;
+
+import duke.exceptions.DukeWrongArgsException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import duke.task.Task;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke {
+
+    static int BOX_WIDTH = 100;
```

Comment:
Perhaps also declare it as a final variable since the naming convention seems to suggest it is a constant?